cli_tools/gce_windows_upgrade/main.go

```diff
@@ -35,6 +35,7 @@ var (
 	sourceOS               = flag.String("source-os", "", fmt.Sprintf("OS version of the source instance to upgrade. Supported values: %v", strings.Join(upgrader.SupportedSourceOSVersions(), ", ")))
 	targetOS               = flag.String("target-os", "", fmt.Sprintf("Version of the OS after upgrade. Supported values: %v", strings.Join(upgrader.SupportedTargetOSVersions(), ", ")))
 	timeout                = flag.String("timeout", "", "Maximum time limit for an upgrade. For example, if the time limit is set to 2h, the upgrade times out after two hours. For more information about time duration formats, see $ gcloud topic datetimes")
+	useStagingInstallMedia = flag.Bool("use-staging-install-media", false, "Use staging install media. This flag is for testing only. Set to true to upgrade with staging windows install media.")
```

Comment:
Even it's only for internal testing purpose, I still don't want it to be abused since it's not really hidden. If the customer used a bad image, their instance can be broken...
Also, for testing staging image, there is only one correct image path. All other paths are wrong.

Reply:
Fair enough.